### ISSUE TITLE
Fill in details for path references

### ISSUE THREAD

**Issue by HeikoTheissen** [xtofs/abstract-csdl#1]
The section on path references is incomplete:

https://github.com/xtofs/abstract-csdl/blob/ada5bfefafe0f57cd6c627501f1d01e6f2abbc3b/acsdl.md?plain=1#L709-L729

It should be called out that a `KeyPropertyPath` not just references a CSDL element, but a path to one, as explained in https://github.com/oasis-tcs/odata-specs/issues/288#issuecomment-2014482536.

**Comment by xtofs** [comment]
I agree that it important that `KeyPropertyPath` as I had to discover this myself with great help from you.

This is why it is in the section `### path reference attribute types` and has the "meta-type" `PathReference`. It is also described as "a path ending in [...]". 

Or are you referring to the fact that the markdown seems to have a bug and doesn't show  anything after "ending in one of the following elements:" ?

**Comment by HeikoTheissen** [comment]
> Or are you referring to the fact that the markdown seems to have a bug and doesn't show anything after "ending in one of the following elements:" ?

Yes.
